lambdas/build-client-oauth-response/src/main/java/uk/gov/di/ipv/core/buildclientoauthresponse/BuildClientOauthResponseHandler.java

```diff
@@ -180,6 +182,20 @@ public Map<String, Object> handleRequest(JourneyRequest input, Context context)
                             auditEventUser,
                             new AuditRestrictedDeviceInformation(input.getDeviceInformation())));
 
+            var isReproveIdentity = clientOAuthSessionItem.getReproveIdentity();
+            if (Boolean.TRUE.equals(isReproveIdentity)
+                    && ipvSessionItem.getVot() != null
+                    && List.of(Vot.P0, Vot.P2).contains(ipvSessionItem.getVot())) {
+                auditService.sendAuditEvent(
+                        AuditEvent.createWithoutDeviceInformation(
+                                AuditEventTypes.IPV_ACCOUNT_INTERVENTION_END,
+                                configService.getSsmParameter(ConfigurationVariable.COMPONENT_ID),
+                                auditEventUser,
+                                AuditExtensionAccountIntervention
+                                        .newReproveIdentityAuditExtensionAccountIntervention(
+                                                ipvSessionItem.getErrorCode() == null)));
```

Comment:
I think this should be checking the VOT. For now P2 = success, anything else = failure.

(However in future this will need to capture P1 as well - cc @blakeyp )

Reply:
Ah ok thanks - should we not check the error code at all then?

Reply:
Happy to leave in the error code check, but I don't think it's needed.

The VOT shouldn't be set to P2 until we've confirmed that they have successfully proved their identity. In practice the only error code you can get after that is a timeout on the success screen - but it doesn't hurt to 'fail safe'.

Reply:
thanks, will leave it as it is then :)